### PR TITLE
Enhance Error Email Specificity with User, Host, and IP Details

### PR DESCRIPTION
**Pull request by Hari-Narayan-dev**

### CODE REVIEW

File: lib/pino-http-send.mjs [hunk 1]
```diff
@@ -10,9 +21,25 @@ const createWriteStream = async function (options) {
...
             for await (let line of source) {
-                const task = fetch(options.url, {
-                    ...options,
-                    body: JSON.stringify({ message: line })
+                const { err = {}, error = {}, req = {} } = JSON.parse(line);
+                const params = { ...req.params, ...req.body };
+                const formParams = new URLSearchParams({
+                    SystemPath: process.cwd(),
+                    RemoteHost: req.remoteAddress || '',
+                    Form: JSON.stringify(params || ''),
```
Comment (durlabhjain) [inline lib/pino-http-send.mjs:29]: Are we sure we are using JSON.stringify from C# or are we creating a key value pair separated by new line?

Same goes with QueryString

File: lib/pino-http-send.mjs [hunk 2]
```diff
@@ -10,9 +21,25 @@ const createWriteStream = async function (options) {
...
+                    QueryString:  JSON.stringify(req.query || ''),
+                    UserAgent: req.headers ? req.headers['user-agent'] : getHeaderValue(req?.rawHeaders, 'User-Agent') || '',
+                    RawUrl: req.url || '',
+                    User: req.Username || '',
+                    AbsoluteUrl: `${req.serverUrl}internal-reporting${req.url}`
```
Comment (durlabhjain) [inline lib/pino-http-send.mjs:34]: Why are we hard coding internal-reporting inframework?

AbsoluteleUrl and RawUrl - check the differences and use that logic

May be we just need 1

**Review by durlabhjain — CHANGES_REQUESTED** [review]

File: lib/pino-http-send.mjs (hunk 1 above)
Comment (Hari-Narayan-dev) [reply lib/pino-http-send.mjs]: JSON.stringify is removed and sending the urlsearchparams as checked in C# we are sending the urlsearchparams

File: lib/pino-http-send.mjs (hunk 2 above)
Comment (Hari-Narayan-dev) [reply lib/pino-http-send.mjs:34]: removed the hard coded internal-reporting in framework and checked we are getting raw url and absolute url both from c#

**Review by durlabhjain — APPROVED** [review]

**Pull request merged**